apps/frontend/src/app/components/insightConfiguration/ConsentsSearchModal.tsx

```diff
@@ -10,37 +11,29 @@ interface IProps {
     addConsent: (consent: IConsent) => void;
 }
 
-export const ConsentsSearchModal = ({
-    open,
-    close,
-    addConsent,
-}: IProps): ReactElement => {
-    // Query API for results
-    // Remove item on click
-    const consents: IConsent[] = [
-        { id: 1, description: 'Samtykker til skjermopptak' },
-    ];
+export const ConsentsSearchModal = ({ open, close, addConsent }: IProps): ReactElement => {
+    const [search, setSearch] = useState('');
+    const { consents, loading, error } = useConsentSearchByDescription(search);
+
+    const handleSearch = (event: ChangeEvent<HTMLInputElement>) => {
+        setSearch(event.target.value);
+    };
 
     return (
         <Modal open={open} onClose={close}>
             <Modal.Content className={style.wrapper}>
-                <SearchField
-                    label="Kriterier"
-                    description={<div>Søk etter samtykker her</div>}>
-                    <SearchField.Input />
+                <SearchField label="Kriterier" description={<div>Søk etter samtykker her</div>}>
```

Comment:
Unsure if this is for the creation of consents, or if searching for consents connected to a person. But if it is for consents connected to a person, i think it instead should just list all consents connected to a person.